driverconfig/powerstore_v240_v121.json

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
Please remove all json files related to powerstore_v210

Reply:
done